modules/contact/include/userobjects/BilinearMixedModeCohesiveZoneModel.h

```diff
@@ -9,26 +9,23 @@
 
 #pragma once
 
-#include "PenaltySimpleCohesiveZoneModel.h"
+#include "CohesiveZoneModelBase.h"
 #include "TwoVector.h"
 
 /**
  * User object that interface pressure resulting from a simple traction separation law.
```

Comment:
I know this isn't your line, but the sentence doesn't make sense. Maybe it means to say
```suggestion
 * User object that computes interface pressure from a simple traction separation law.
```